src/Elastic.Apm/Config/AbstractConfigurationReader.cs

```diff
@@ -276,7 +276,7 @@ protected bool ParseTraceContextIgnoreSampledFalse(ConfigurationKeyValue kv)
 			// ReSharper disable once SimplifyConditionalTernaryExpression
 			return bool.TryParse(kv.Value, out var value) ? value : DefaultValues.TraceContextIgnoreSampledFalse;
 		}
-
+		protected bool ParseUseWindowsCredentials(ConfigurationKeyValue kv) => ParseBoolOption(kv, DefaultValues.CaptureHeaders, "UseWindowsCredentials");
```

Comment:
Sorry for the nit ... can I ask you to add a blank line here for consistency reasons.